### PR TITLE
Update API to lingui v3

### PR DESCRIPTION
**Pull request by sleepycat**
This required removing an option to provide a value for missing translations which seemed to bork things, so there are a lot of files ended up getting touched.

### CODE REVIEW

File: api-js/src/organization/objects/organization-summary.js [hunk 1]
```diff
@@ -23,12 +22,12 @@ export const organizationSummaryType = new GraphQLObjectType({
 
         const categories = [
           {
-            name: i18n._(t`pass`),
+            name: i18n._('pass'),
```
Comment (nsdeschenes) [inline api-js/src/organization/objects/organization-summary.js:25]: After removing the `t` macro, then running `npm run lin-clean` the translations are then removed from `.po` files.

Screenshot from 2021-01-21 07-09-14

**Review by nsdeschenes — CHANGES_REQUESTED** [review]

File: api-js/src/organization/objects/organization-summary.js (hunk 1 above)
Comment (sleepycat) [reply api-js/src/organization/objects/organization-summary.js]: Good catch! I've updated it to use `defineMessage` to define a translatable string that lingui recognizes and extracts. Take another look and tell me what you think.

File: api-js/package.json [hunk 2]
```diff
@@ -8,26 +8,26 @@
   "author": "nsdeschenes",
   "license": "MIT",
   "scripts": {
-    "build": "npm run clean && babel ./src --out-dir dist/src --ignore 'src/**/*.spec.js','src/**/*.test.js' && babel index.js migrations.js --out-dir dist/",
-    "start": "node -r dotenv-safe/config ./dist/index.js",
-    "dev": "nodemon --exec babel-node -r dotenv-safe/config index.js",
+    "build": "npm run clean && babel ./src --out-dir dist/src --ignore 'src/**/*.spec.js','src/**/*.test.js' && cp ./index.js dist/ && cp ./migrations.js dist/",
+    "start": "node ./dist/index.js",
+    "dev": "nodemon --exec babel-node index.js",
```
[the remark this answers is not in the document]
Comment (sleepycat) [reply api-js/package.json]: Reverted!

File: api-js/package.json [hunk 3]
```diff
@@ -46,7 +46,7 @@
     "moment": "^2.29.1",
     "notifications-node-client": "^5.1.0",
     "psl": "^1.8.0",
-    "url-slug": "^3.0.1",
+    "url-slug": "^2.3.2",
```
[the remark this answers is not in the document]
Comment (sleepycat) [reply api-js/package.json]: Reverted!

**Review by nsdeschenes — APPROVED** [review]

**Pull request merged**